apps/ws/src/socket/services/ws-server-health-indicator.service.ts

```diff
@@ -7,16 +7,21 @@ import { WSGateway } from '../ws.gateway';
 
 @Injectable()
 export class WSServerHealthIndicator extends HealthIndicator implements IHealthIndicator {
-  private INDICATOR_KEY = 'ws-server';
+  private static KEY = 'ws-server';
 
   constructor(private wsGateway: WSGateway) {
     super();
   }
 
   async isHealthy(): Promise<HealthIndicatorResult> {
-    const status = !!this.wsGateway.server;
+    const isHealthy = !!this.wsGateway.server;
+    const result = this.getStatus(WSServerHealthIndicator.KEY, isHealthy);
```

Comment:
✏ **suggestion:** this function seems to need to be called only when `isHealthy` is `true`, so maybe consider moving it under the if statement. Same in the other places.

Reply:
The result is also used in the thrown error.

Reply:
ahh sorry, didn't noticed that :)